region/lowernorfair/east/Lower Norfair Fireflea Room.json

```diff
@@ -407,6 +407,31 @@
       "flashSuitChecked": true,
       "note": "Place a Power Bomb then exit G-Mode to kill the Fune."
     },
+    {
+      "link": [2, 2],
+      "name": "Gain Flash Suit (Fune Dead, Spikesuit)",
+      "requires": [
+        {"obstaclesCleared": ["A"]},
+        {"canShineCharge": {"usedTiles": 20, "gentleDownTiles": 2, "openEnd": 1}},
+        {"spikeHits": 1},
+        {"or": [
+          "h_SpikeSuitSpikeHitLeniency",
+          {"and": [
+            {"resourceCapacity": [{"type": "PowerBomb", "count": 1}]},
+            {"resetRoom": {"nodes": [2]}},
+            {"or": [
+              "SpaceJump",
+              "HiJump",
+              "canPreciseWalljump"
```

Comment:
```suggestion
              "canPreciseWalljump",
              "canSpringBallJumpMidAir",
              "canIBJ"
```

Reply:
Spring Ball jump and IBJ were considered too slow to be reasonable as a way to farm between attempts. They're also left out of the "Fireflea and Fune Farm" strat. I don't know though; maybe they could be ok to add? @osse101 What do you think?

Reply:
In my vanilla map randomizer experience, farming firefleas takes much longer than you would think.  And gets a lot more complicated if you want all the drops instead of 2health+1PB.
IBJ or springballjump would be fine for the ridley claw jump.  But you have to do another speedyjump into itemTech to reach the Fune.  And doing both is too much.  But if logic is officially only requiring 1 trip or 1 and a bit of fireflea farming, you could do that.
Spikesuits are not that likely to hit in 3 tries but I wouldn't want to require more effective health either.

Reply:
I was thinking that the spike suit leniency is only 2 extra spike hits, so that would mean that you're not needing to farm a lot of Energy. But you do need to use a PB to get down there and reset and use another PB to turn the lights back on. I think IBJ is probably not worth including, but spring may be?

Reply:
With springball its a trickydash jump, and that's not fun in the dark.  I'd be more ok with `canJumpIntoIBJ` getting up.  Pausing a bunch is just less fun than IBJing too.
The first PB gives two Fune drops and that pays for itself.  The farming sequence is pretty slow though.
It's `canSpikesuit` level of difficulty & movement so they can make it work, but I'd rather require the extra health.  Also, it feels bad to only get one try at the spikesuit per shortcharge.
It looks like the way it plays out, you make one attempt and then farm 90% of your health back and then resetRoom.